arcgis-ios-sdk-samples/Layers/Group layers/GroupLayersSectionView.xib

```diff
@@ -0,0 +1,50 @@
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="14460.31" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES">
+    <device id="retina4_7" orientation="portrait">
+        <adaptation id="fullscreen"/>
+    </device>
+    <dependencies>
+        <deployment identifier="iOS"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="14460.20"/>
+        <capability name="Safe area layout guides" minToolsVersion="9.0"/>
+        <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
+    </dependencies>
+    <objects>
+        <placeholder placeholderIdentifier="IBFilesOwner" id="-1" userLabel="File's Owner" customClass="LayersTableViewController" customModule="ArcGIS_Runtime_SDK_Samples" customModuleProvider="target"/>
+        <placeholder placeholderIdentifier="IBFirstResponder" id="-2" customClass="UIResponder"/>
+        <view contentMode="scaleToFill" id="XjP-9q-BD7" customClass="GroupLayersSectionView" customModule="ArcGIS_Runtime_SDK_Samples" customModuleProvider="target">
+            <rect key="frame" x="0.0" y="0.0" width="375" height="44"/>
+            <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+            <subviews>
+                <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" horizontalCompressionResistancePriority="749" text="Label" textAlignment="natural" lineBreakMode="wordWrap" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="10R-ek-huY">
+                    <rect key="frame" x="16" y="5" width="300" height="34"/>
+                    <fontDescription key="fontDescription" type="boldSystem" pointSize="17"/>
+                    <nil key="textColor"/>
+                    <nil key="highlightedColor"/>
+                </label>
+                <switch opaque="NO" contentMode="scaleToFill" horizontalHuggingPriority="750" verticalHuggingPriority="750" contentHorizontalAlignment="center" contentVerticalAlignment="center" on="YES" translatesAutoresizingMaskIntoConstraints="NO" id="9ku-4w-xzb">
+                    <rect key="frame" x="321" y="5" width="51" height="34"/>
+                    <connections>
+                        <action selector="toggleSwitch:" destination="-2" eventType="valueChanged" id="ndi-hb-abW"/>
+                    </connections>
+                </switch>
+            </subviews>
+            <constraints>
+                <constraint firstItem="10R-ek-huY" firstAttribute="top" secondItem="XjP-9q-BD7" secondAttribute="top" constant="5" id="1AJ-iz-MjX"/>
+                <constraint firstItem="9ku-4w-xzb" firstAttribute="leading" secondItem="10R-ek-huY" secondAttribute="trailing" constant="5" id="Lt2-qq-1br"/>
```

Comment:
Setting a fixed distance like this is fairly common, but it has one drawback: it makes the label wider than it needs to be. We can correct that by making this a `greaterThanOrEqual` relation. It allows a label to be as small as needed to fit its text while preventing the label from encroaching on the switch.

![image](https://user-images.githubusercontent.com/2257493/56171075-d84b6c00-5f98-11e9-9040-085f20d89f77.png)

Before:

![image](https://user-images.githubusercontent.com/2257493/56171135-0af56480-5f99-11e9-86d3-08faa4d1ac40.png)

After:

![image](https://user-images.githubusercontent.com/2257493/56171116-fdd87580-5f98-11e9-956b-70df3f6d415d.png)

Same with the table view cell.